### ISSUE TITLE
Problem on the "vote_reg_loss"

### ISSUE THREAD

**Issue by NUAAXQ** [qq456cvb/VoteNet#11]
Hi, I notice that in the model.py line 84 the vote_reg_loss is defined as:

vote_reg_loss = tf.reduce_mean(tf.norm(votes_xyz - bboxes_xyz_votes_gt, ord=1, axis=-1) * tf.cast(surface_ind, tf.float32), name='vote_reg_loss')

where
votes = tf.concat([seeds_xyz, seeds_points], 2) + "offset"
votes_xyz = votes[:, :, :3]

However, in the paper, the vote_reg_loss is defined as the difference between the "offset" (Δx, vote_xyz - seed_xyz, equation (1) in the paper), not the difference between the vote_xyz and the bboxes_center.

So, is there some problem with the vote_reg_loss in the code?

**Comment by NUAAXQ** [comment]
I think the correct vote_reg_loss should be "offset-dist2center" according to the equation (1) in the paper. Is that right?

**Comment by qq456cvb** [comment]
We have `votes_xyz = offset + seeds_xyz` (L60), `seeds_xyz = bboxes_xyz + dist2center` (L62);
so that `offset + dist2center = votes_xyz - seeds_xyz + seeds_xyz - bboxes_xyz = votes_xyz - bboxes_xyz`. They are equivalent.

**Comment by NUAAXQ** [comment]
@qq456cvb the loss is "offset - dist2center", not "offset + dist2center". So, according to line60 and line 62, offset - dist2center = votes_xyz - seeds_xyz - seeds_xyz + bboxes_xyz.

**Comment by qq456cvb** [comment]
We mean the same thing, the delta in the paper is the opposite number of dist2center in the code.

**Comment by NUAAXQ** [comment]
@qq456cvb  ok, I got it. 
The problem is the dist2center in the code is the opposite numder in the paper. So there is no conflict at all. Your code is right.
Thank you!
